Loop/Managers/LoopDataManager.swift

```diff
@@ -184,12 +199,15 @@ class LoopDataManager {
 
      - parameter resultsHandler: A closure called once the values have been retrieved. The closure takes the following arguments:
         - predictedGlucose:     The calculated timeline of predicted glucose values
+        - insulinEffect:        The predicted effect of insulin
+        - carbEffect:           The predicted effect of carbohydrates
```

Comment:
Not sure I follow with the "only [x] were considered"?

I'm good with merging this PR if you are, which would mean reconciling this docstring and reverting the `import` line at the top. It sounds like there's still some munging of the carb/insulin effect lines for Nightscout, though ideally you can sort that processing out in the cloud. Just let me know what you'd like to do.